### ISSUE TITLE
No field INSTANCE 

### ISSUE THREAD

**Issue by loveleopard** [Adyen/adyen-java-api-library#576]
**Java version**: x.y.z
**Library version**: 15.0.2
**Description**

2021-06-23 17:35:32.809 27878-27946/com.neolix.goggo E/AndroidRuntime: FATAL EXCEPTION: Thread-2
    Process: com.neolix.goggo, PID: 27878
    java.lang.NoSuchFieldError: No field INSTANCE of type Lorg/apache/http/conn/ssl/AllowAllHostnameVerifier; in class Lorg/apache/http/conn/ssl/AllowAllHostnameVerifier; or its superclasses (declaration of 'org.apache.http.conn.ssl.AllowAllHostnameVerifier' appears in /system/framework/framework.jar:classes2.dex)
        at org.apache.http.conn.ssl.SSLConnectionSocketFactory.<clinit>(SSLConnectionSocketFactory.java:151)
        at com.adyen.httpclient.AdyenHttpClient.createCloseableHttpClient(AdyenHttpClient.java:208)
        at com.adyen.httpclient.AdyenHttpClient.request(AdyenHttpClient.java:105)
        at com.adyen.httpclient.AdyenHttpClient.request(AdyenHttpClient.java:100)
        at com.adyen.service.Resource.request(Resource.java:97)
        at com.adyen.service.Resource.request(Resource.java:63)
        at com.adyen.service.TerminalLocalAPI.request(TerminalLocalAPI.java:66)
        at com.neolix.goggo.GoggoPayManager.payment(GoggoPayManager.java:87)
        at com.neolix.goggo.MainActivity$1.run(MainActivity.java:24)
        at java.lang.Thread.run(Thread.java:761)

**Comment by JasonBuffer** [comment]
@loveleopard  hi,I also encountered the same problem, have you found a solution?

**Comment by tjshakespeare** [comment]
We are also having this problem since upgrading our library version to 15+. Reverting back to version 14.0.0 resolved it for us. But we really want some of the improvements in version 15+, so isn't really viable long term.

I think it due to the move to using the Apache library in 15.0.0, but I have noticed that there has recently been a PR that's made it through to develop in this sort of area. Fingers cross the next release solves this problem and will be available soon.

**Comment by tjshakespeare** [comment]
@loveleopard @JasonBuffer 

Just to add on what I mentioned before, I forked and checked out the current develop branch, built it and included the .jar file into my project instead of adding the dependency like I normally would. I did this to see if the changes that has made its way through would work and I am no longer getting this issue. It seems to be whatever had been changed in develop recently has solved this problem. 

Now just need to wait on the next release that contains this fix. Does anybody have an idea on when we may be able to expect the next release build available? Usually they are quite regular, once or twice a month so I guess we wont have to wait long for this?
